js/jquery.nice-select.js

```diff
@@ -31,9 +32,29 @@
             ($(this).is(':selected') ? ' selected' : '') +
             ($(this).is(':disabled') ? ' disabled' : '') +
             '" data-value="' + $(this).val() + '"' +
-            (display ? ' data-display="' + display : '') + '"' +
+            (display ? ' data-display="' + display + '"' : '') +
             '>' + $(this).text() + '</li>');
         });
+        
+        
+        /*
+           Events
+        */
+        
+        // Updates the dropdown after setting the value manually
+        select.on('update', function() {
```

Comment:
`update` is not a valid event on select boxes it should be `change`? That said, I had a bit of play with this and found that the change event doesn't get trigger when attempting to update the selected item in code, it requires browser interaction and if you trigger it manually you get stuck in an infinite loop and crashes my browser tab.